lib/smart_proxy_dynflow_core/core.rb

```diff
@@ -73,6 +76,14 @@ def ensure_initialized
         @instance
       end
 
+      def plugin_matchers
+        @matchers ||= []
+      end
+
+      def register_plugin_matchers(matchers)
```

Comment:
Similarly to `plugin_matchers`, thinking about improving the naming. What about `register_silencer_matchers`, as we don't really care if it's from plugins, but it should be clean it's about silencing